contracts/Governance/AccessControlled.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.13;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/access/Ownable2StepUpgradeable.sol";
+
+import "./IAccessControlManager.sol";
+
+abstract contract AccessControlled is Initializable, Ownable2StepUpgradeable {
```

Comment:
Would it make sense to move (not now, in the future) this abstract class to the new Governance package?

Reply:
Makes sense IMO

Reply:
VEN-1213 created